core/sql/exp/ExpLOBaccess.cpp

```diff
@@ -2280,8 +2280,10 @@ Ex_Lob_Error ExLobsOper (
 	  return LOB_GLOB_PTR_ERROR;
 	}
     }
-
-  if ((globPtr != NULL) && (operation != Lob_Init))
+  if (globPtr != NULL)
+  {
+    lobGlobals = (ExLobGlobals *)globPtr;
+    if ((operation != Lob_Init) && (operation != Lob_Cleanup))
```

Comment:
Thanks @DaveBirdsall.  Line 2285 is needed. I will remove line 2288 when I get a chance to submit something else.